api/v1alpha1/openapi.yaml

```diff
@@ -2050,6 +2054,31 @@ components:
               additionalProperties: true
           required:
           - inline
+    HttpConfigProviderSpec:
+      allOf:
+        - $ref: '#/components/schemas/GenericConfigSpec'
+        - type: object
+          properties:
+            httpRef:
+              type: object
+              properties:
+                repository:
+                  type: string
+                  description: |
+                    The name of the repository resource to use as the sync source
```

Comment:
Can we clarify this description? Is this a URL? If we are going to bolt on a type to this pattern that really doesn't fit IMO the docs should be very clear.

```suggestion
                    The URL of the HTTP resource that is used as a source for configuration settings
```

Reply:
IIUC, in this case the description is correct, as this is the actual name (ID) of a repository in FlightCTL.

But I can see the confusion because I had a related one here https://github.com/flightctl/flightctl/pull/359/files#diff-f39cc8ce70665fac31d92fec81913428f186e708cb2dce8a1100cd9ea89be690R1876

So my suggestion would be to rename the "repo" prop in the repository specs.